detox/detox.d.ts

```diff
@@ -1608,14 +1672,67 @@ declare global {
           toExist(): R;
         }
 
+        interface SecuredWebElement extends SecuredWebElementActions {
+        }
+
+        interface SecuredWebElementActions {
+            /**
+             * (iOS Only) Tap on a secured web element.
+             * @note Secured-Web APIs are still in experimental phase and are subject to changes in the near future.
+             * @example await web.element(by.web.type('textField')).asSecured().tap();
+             */
+            tap(): Promise<void>;
+
+            /**
+             * (iOS Only) Type text into a web element.
+             * @param text to type
+             * @param isContentEditable whether the element is content-editable, default is false. Ignored on iOS.
+             * @note Secured-Web APIs are still in experimental phase and are subject to changes in the near future.
+             * @example await web.element(by.web.type('textField')).asSecured().typeText('passcode');
+             */
+            typeText(text: string, isContentEditable: boolean): Promise<void>;
+
+            /**
+             * (iOS Only) Replaces the input content with the new text.
+             * @param text to replace with the old content.
+             * @note Secured-Web APIs are still in experimental phase and are subject to changes in the near future.
+             * @example await web.element(by.web.type('textField')).asSecured().replaceText('passcode');
+             */
+            replaceText(text: string): Promise<void>;
+
+            /**
+             * (iOS Only) Clears the input content.
+             * @note On Android, not working for content-editable elements.
+             * @note Secured-Web APIs are still in experimental phase and are subject to changes in the near future.
+             * @example await web.element(by.web.type('textField')).asSecured().clearText();
+             */
+            clearText(): Promise<void>;
+        }
+
         interface IndexableWebElement extends WebElement {
             /**
              * Choose from multiple elements matching the same matcher using index.
-             * @example await web.element(by.web.tag('p')).atIndex(2).tap();
+             * @example await web.element(by.web.tag('p')).asSecured().atIndex(2).tap();
```

Comment:
What's the sense to use `asSecured()` for a matcher (`by.web.tag`) which likely won't work for a secured context?

Reply:
no sense, this is an error obviously, I'll replace it with by.type / label. good catch